crates/ibc-apps/ics20-transfer/types/src/serializers.rs

```diff
@@ -0,0 +1,25 @@
+use core::fmt::Display;
+use core::str::FromStr;
+
+use ibc::prelude::*;
+use serde::{de, Deserialize, Deserializer, Serializer};
+
+// Note: used String version (slower + heap) instead of str,
+// because both str ser/de hit some kind of f64/f32 case when compiled into wasm
+// and fails to be validated f32/f64 wasm runtimes
```

Comment:
```suggestion
// Note: This method serializes to a String instead of a str
// in order to avoid a wasm compilation issue. Specifically,
// str (de)serialization hits some kind of f64/f32 case
// when compiled into wasm, but this fails validation on 
// f32/f64 wasm runtimes.
```